src/components/Swatch/Colors.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Fragment } from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { ColorCategory, COLORS } from '../../colors';
+
+import { SwatchBlock } from './SwatchBlock';
+
+const transformColors = (colors: ColorCategory) => {
+	return Object.entries(colors).map(([name, value]) => ({
+		name,
+		value,
+	}));
+};
+
+storiesOf('Colors', module)
+	.add('Grays', () => <SwatchBlock title="Grays" colors={transformColors(COLORS.GRAYS)} />)
```

Comment:
generalize this?
als we dan een categorie toevoegen wordt dit automatisch getest

Reply:
Fixed